### PR TITLE
Added containers.md file

### PR DESCRIPTION
**Pull request by lslezak**
Added a short document about using containers for system management.

### CODE REVIEW

File: containers.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+## Disadvantages
+
+Unfortunately there are also some disadvantages:
+
+- More complex architecture
+- We will need to build, distribute and maintain the containers
+  (in addition to the RPM packages)
+- It's a question whether all management tasks can be done from a container,
+  maybe for some tasks we might need a dedicates service or special support
```
Comment (ancorgs) [inline containers.md]: Typo: dedicated (I assume)

File: containers.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  in the host system
+
+## New Problems
+
+There are also some completely new problems, specific to the containers environment.
+
+In the current SLE we might be pretty sure that in e.g. in SLE-15-SP3 you use the
```
Comment (ancorgs) [inline containers.md]: "in eg. in" is kind of hard to read

**Comment by ancorgs** [comment]
The document mention several docker options. But based on the several discussions around ALP I would be surprised if ALP uses docker as container runtime. Are similar options available for podman, CRI-O and/or RunC?

**Comment by lslezak** [comment]
I'm not familiar with any of these tools, but after quickly reading the podman documentation (https://docs.podman.io/en/latest/markdown/podman-run.1.html) it seems that most of the docker options are reused there. (That actually makes sense, you want to allow easy transition to your new tool...)

**Review by ancorgs — APPROVED** [review]
LGTM

**Pull request merged**